features/product/managing_products/filtering_products_by_channel.feature

```diff
@@ -0,0 +1,33 @@
+@managing_products
+Feature: Filtering products by a channel
+    In order to filter products by a specific channel
```

Comment:
🚨 "In order to X, I want to X" is a pattern that doesn't really address the need :)

```suggestion
    In order to get a more focused view while managing some channel's products
```